packages/client/ui/react-native/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -200,7 +215,10 @@ export function CrossmintAuthProvider({
                 : urlOrOneTimeSecret;
 
             if (oneTimeSecret != null) {
-                return await crossmintAuth.handleRefreshAuthMaterial(oneTimeSecret);
+                setInProgress(true);
```

Comment:
see https://github.com/Crossmint/crossmint-sdk/pull/1055/files#r2047670155 if we use `finally` block we might not need to handle `inProgress` state here